deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -116,18 +116,26 @@ rules:
   - create
   - update
   - delete
-- apiGroups:
-  - ceph.rook.io
-  resources:
-  - "*"
-  verbs:
-  - "*"
-- apiGroups:
-  - rook.io
-  resources:
-  - "*"
-  verbs:
-  - "*"
+# The Rook operator must be able to watch all ceph.rook.io resources to reconcile them.
+- apiGroups: ["ceph.rook.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
+# The Rook operator must be able to update all ceph.rook.io resources to add finalizers to them.
+# There is no current ability (Dec. 2021) to update finalizers separately from the rest of the
+# object like how different permissions can be given for the status sub-resource.
+# When finalizers become an independently-updateable sub-resource, we should revisit this.
+# The "finalizers" permission may need to be strictly given for clusters where
+# OwnerReferencesPermissionEnforcement is enabled so that Rook can set blockOwnerDeletion on
+# resources owned by Rook CRs. See more:
+# https://kubernetes.io/docs/reference/access-authn-authz/_print/#ownerreferencespermissionenforcement
+- apiGroups: ["ceph.rook.io"]
+  resources: ["*", "*/finalizers"]
```

Comment:
I'm trying to understand here the difference between
`  resources: ["*", "*/finalizers"]` vs `  resources: ["*/finalizers"]`

Reply:
This role item ideally allows rook to update finalizers on the resource without giving update permissions to the spec. However, that's not currently how the `*/finalizers` permission works. The comment block above attempts to explain it, but it's really hard to summarize.